src/code/containers/challenge-container-selector.js

```diff
@@ -24,7 +24,7 @@ function mapContainerNameToContainer(containerName) {
 class ChallengeContainerSelector extends Component {
 
   static propTypes = {
-    authoring: PropTypes.object.isRequired,
+    authoring: PropTypes.array.isRequired,
```

Comment:
One additional thought on this topic: seems like an opportunity for an automated test. It would be more of an acceptance test than a unit test, but one could imagine instantiating all challenges with traps for propTypes warnings and other such failures. Details on how to go about it at [Make React PropType warnings throw errors with enzyme.js + sinon.js + mocha.js](https://gist.github.com/scmx/d98cc058a7c3dfef7890) and [Testing React PropTypes with sinon](http://stackoverflow.com/questions/40859389/testing-react-proptypes-with-sinon). Not something to hold up this PR, but worth thinking about moving forward.